commit 654065dc30188f660f5f29881d8d3314074bea8f (refs/changes/84/3284/3)
Author: Kody A Kantor <kody.kantor@gmail.com>
Date:   2018-01-31T22:58:32+00:00 (1 year, 8 months ago)
    
    joyent/manta-thoth#169 THOTH_USER environment variable should be documented
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/README.md b/README.md
index 2858f88..1e17a99 100644
--- a/README.md
+++ b/README.md
@@ -21,6 +21,12 @@ need to set Manta environment variables that match your Joyent Manta account:
     $ export MANTA_URL=https://us-east.manta.joyent.com
     $ export MANTA_USER=bcantrill
 
+You may also need to set your `THOTH_USER` environment variable if you are using
+a shared Thoth installation. For example, if your shared Thoth installation
+uses the 'thoth' Manta user:
+
+    $ export THOTH_USER=thoth
+
 # Running Thoth
 
 ## Introduction
@@ -371,7 +377,7 @@ as `root`):
 ## License
 
 The MIT License (MIT)
-Copyright (c) 2013 Joyent
+Copyright (c) 2018 Joyent
 
 Permission is hereby granted, free of charge, to any person obtaining a copy of
 this software and associated documentation files (the "Software"), to deal in
